ppx_cstruct.opam

```diff
@@ -23,5 +23,6 @@ depends: [
   "ocaml-migrate-parsetree"
   "ppx_driver"    {test & >= "v0.9.0"}
   "ppx_sexp_conv" {test}
+  "cstruct-unix"  {test}
```

Comment:
where in travis.yml needs that `TEST=true` to be?  this didn't use to be the case, or was it?  I remember having seen travis executing tests without it!?!?